docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
 
 ## Overview
 
+Once your data is cleaned, you will need to get it out of OpenRefine and into the system of your choice. OpenRefine outputs a number of file formats, can upload your data directly into Google Sheets, and can export directly to Wikidata. 
+
+You can also export your full project data so that it can be opened by someone else using OpenRefine (or yourself, on another computer).
 
 ## Export data
 
 Note you will only export data in the current view - that is, with current filters and facets applied. 
 
+To export from a project, click the Export button at the top right corner and pick the format you want. You options are:
 
-
-*   TSV/CSV
-*   HTML table
+*   Tab-separated value (TSV) or Comma-separated value (CSV)
+*   HTML-formatted table
 *   Excel (XLS or XLSX)
 *   ODF spreadsheet
-*   Google Sheets \
+*   Upload to Google Sheets (requires Google account authorization)
+*   [Custom tabular exporter](#custom-tabular-exporter)
+*   [SQL statement exporter](#sql-statement-exporter)
+*   [Templating exporter](#templating-exporter)
 
-*   Custom tabular export
-*   SQL
-*   Templating export \
+You can also [export reconciled data to Wikidata](wikidata#editing-wikidata-with-openrefine), or save your schema for future use with other OpenRefine projects:
 
 *   Upload edits to Wikidata
 *   Export to QuickStatement
 *   Export Wikidata schema
 
+### Custom tabular exporter
+
+![A screenshot of the custom tabular content tab.](/img/custom-tabular-exporter.png)
+
+With the custom tabular exporter, you can choose which of your data to export, the separator you wish to use, and whether you'd like to download it to your computer or upload it into a Google Sheet. 
+
+You can drag and drop the columns appearing in the column list to reorder the output. The options for reconciled and date data are applied to each column individually. 
+
+This exporter is especially useful with reconciled data, as you can choose whether you wish to output the cells' original values, the matched values, or the matched IDs. Ouputting “match entity's name”, “matched entity's ID”, or “cell's content” will output, respectively, the contents of `cell.recon.match.name`, `cell.recon.match.id`, and `cell.value`. 
+
+“Output nothing for unmatched cells” will export empty cells for both newly-created matches and cells with no chosen matches. “Link to matched entity's page” will produce hyperlinked text in an HTML table output, but have no change in other formats.
```

Comment:
Do we need to say that for "Output nothing..." option what happens for matched cells?

In second sentence I think "have no effect" rather than "have no change" would be better phrasing

Reply:
"Output nothing" will do whatever you've chosen above it for matched cells. Or do you mean something else?